packages/patternfly-4/react-core/package.json

```diff
@@ -46,10 +46,10 @@
     "@patternfly/react-tokens": "^1.0.0"
   },
   "devDependencies": {
-    "@patternfly/patternfly-next": "1.0.44",
+    "@patternfly/patternfly-next": "1.0.46",
```

Comment:
This number needs to match throughout all of the packages (or the build will fail, as it is now).

Reply:
Fixed :)